.ci/ci.yml

```diff
@@ -64,6 +69,20 @@ resources:
         access_token: ((github-account.password))
         label: autogenerated
 
+    - name: gcp-bucket
+      type: gcs-resource
+      source:
+        bucket: ((gcp-bucket))
+        json_key: ((gcp-bucket-json-key))
+        regexp: dist/terraform-provider-google.(*)
+
+    - name: night-trigger
+      type: time
+      source:
```

Comment:
You'll want "interval: 1h", also.

Reply:
In the resource docs says "Trigger once within time range" and it doesn't have the interval var, could you explain it to me, thanks!

Reply:
I was misreading the code for `time-resource` - you got it right, no need to make this change.  :)